Artifacts/windows-add-local-admin/artifact.ps1

```diff
@@ -0,0 +1,137 @@
+[CmdletBinding()]       
+param
+(
+    [Parameter(Mandatory = $true)]
+    [string] $DomainJoinUsername
+)
+
+##############################
+# Function returns the localized group name for the local administrator group
+function Get-LocalAdminGroupName ()
+{
+    #well known SID for the Administrators group
+    ([wmi]"Win32_SID.SID='S-1-5-32-544'").AccountName
+}
+
+#############################
+# Function to find the members of a local machine group, usually administrators.
+# Works around the cahnge in call pattern InvokeMember
+function Get-LocalGroupMembersPs3to5 ([Parameter(Mandatory = $true)] [string] $GroupName)
```

Comment:
Move all parameters to a `param` section inside the function, for all functions. For example,
```
function Get-LocalGroupMembersPs3to5
{
    param(
        ([Parameter(Mandatory = $true)]
        [string] $GroupName
    )
    ...
}
```